portal-web/docroot/html/taglib/ui/search_iterator/bottom.jspf

```diff
@@ -105,6 +107,8 @@
 
 						var syncScrollingContainers = function(event) {
 							fixedHeaderScrollingContainer.scrollLeft = horizontalScrollingContainer.scrollLeft;
+
+							innerWrapperContainer.style.width = tableResponsiveContainer.getBoundingClientRect().width + 'px';
 						}
```

Comment:
This seems to be ok from the perspective of CSP even in presence of `style-src-attr` directive, as explained in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-attr:

> Style properties that are set directly on the element's [style](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style) property will not be blocked, allowing users to safely manipulate styles via JavaScript